examples/plot_choose_thumbnail.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 """
+.. _choose-thumbnail-example:
```

Comment:
Quickly grepping, it seems like you want to remove this line and use `:ref:sphx_glr_auto_examples_plot_choose_thumbnail.py` where you need to reference this example.